app/views/admin/loans/_questions.html.slim

```diff
@@ -4,16 +4,12 @@ section.questionnaires
   / Switch at top allowing toggling between criteria and post analysis
   .filter-switch.centered-text
     .btn-group
-      = link_to t('loan.criteria'),
-        admin_loan_tab_path(@loan.id, filter: "loan_criteria", tab: "questions"),
-        class: "btn btn-default #{'active' if @attrib == 'loan_criteria'}",
-        data: {filter: "loan_criteria"}
-      = link_to t('loan.post_analysis'),
-        admin_loan_tab_path(@loan.id, filter: "loan_post_analysis", tab: "questions"),
-        class: "btn btn-default #{'active' if @attrib == 'loan_post_analysis'}",
-        data: {filter: "loan_post_analysis"}
+      - @question_sets.each do |qset|
```

Comment:
great, glad to have one less thing (The attr) that has to be 'threaded through' a request